source/Timezones.mc

```diff
@@ -0,0 +1,114 @@
+using Toybox.WatchUi as Ui;
+using Toybox.System as Sys;
+using Toybox.Application as App;
+using Toybox.Graphics as Gfx;
+using Toybox.Time.Gregorian as Calendar;
+
+(:timezones)
+class Timezones extends Ui.Drawable {
+
+	var utcOffset = new Time.Duration(-Sys.getClockTime().timeZoneOffset);
```

Comment:
Some mixed naming conventions here: `utc` vs `UTC_`, `tz` vs `TimeZones`, not all have `m` prefix.